src/config.rs

```diff
@@ -48,6 +48,7 @@ pub(crate) struct Config {
     pub(crate) merge_conflicts: Option<MergeConflictConfig>,
     pub(crate) bot_pull_requests: Option<BotPullRequests>,
     pub(crate) rendered_link: Option<RenderedLinkConfig>,
+    pub(crate) relink: Option<RelinkConfig>,
```

Comment:
I don't want to bikeshed too much, but this name is quite generic. Maybe something like `relink_issue_links`, `canonicalize_issue_links` or `normalize_issue_links`?

Reply:
I agree, `relink` is a bit too generic, I think canonicalize is right word so I went with your suggestion.